docs/api/buckets.rst

```diff
@@ -300,3 +300,99 @@ Retrieving all buckets
                 }
             ]
         }
+
+
+.. buckets-default-id:
+
+Personal bucket «default»
```

Comment:
Well it is explained shortly here: https://github.com/Kinto/kinto/pull/257/files#diff-faad629fc7ad04c55c32e64e7a2d19e8R19